docs/tech/https.md

```diff
@@ -0,0 +1,118 @@
+# HTTPS
+
+[HTTPS](https://en.wikipedia.org/wiki/HTTPS) secures information when it's
+being transmitted over the internet. Everybody should use HTTPS, but we especially
+must use HTTPS because of the sensitive data we transmit.
+
+## Compatibility
+
+Some of this will be trial and error. Some settings can be tweaked on our side but
+some particularly old devices/software will just never work with our system.
+
+It's difficult to anticipate what devices are out there being used in large numbers,
+and what browsers are being used on those devices. In a number of cases devices with
+old operating systems (like Windows XP, old Android and old BlackBerry OS) will not
+be compatible with our HTTPS setup when using the default/built-in browser. If users
+have a recent version of a commonly used browser like Google Chrome, Mozilla Firefox,
+or Opera Browser installed then that should be compatible.
+
+Users accessing an HTTPS site via a recent version of Opera Mini should not have any
+compatibility issues on any platform but the effectiveness of the encrypted connection
+is reduced as the connection is decrypted at Opera's servers.
+
+### Server Name Indication (SNI) compatibility
+
+The first and most important feature that any device connecting to an HTTPS site on our
+cluster will need is Server Name Indication (SNI) support. This, unfortunately, cannot
+be worked around and is a strict requirement born out of the fundamental design of the
+cluster. Please see [the Wikipedia article](https://en.wikipedia.org/wiki/Server_Name_Indication#Suppor)
+for full details.
+
+The native/built-in browsers on the following mobile devices do not support SNI (although
+it may be possible for users to install a browser that does support SNI):
+
+- Android 2.3 "Gingerbread" and earlier
+- Nokia Symbian OS
+- BlackBerry OS 7.1 and earlier
+
+The following browsers on desktop devices do not support SNI:
+
+- Internet Explorer on Windows XP
+
+It's also important to note that the following Python software we use does not support SNI:
+
+- Any Python software running on Python < 2.7.9 (and note that the version of Python 2.7 provided
+  with Ubuntu 14.04 is 2.7.6. Also note that our Python Docker containers will always run the
+  latest versions of Python, currently 2.7.12 for Python 2 containers)
+- Vumi < 0.6.15
+- Twisted < 14.0.0
+
+### Transport Layer Security (TLS) compatibility
+
+The load balancer is currently only configured to support recent versions of
+Transport Layer Security (TLS), namely TLS 1.1 and 1.2. TLS 1.0 support is currently disabled.
```

Comment:
I just ran [a test against TuneMe](https://www.ssllabs.com/ssltest/analyze.html?d=tuneme.org&s=197.189.237.42&latest) and it says that only 1.2 is enabled. Is this wrong for some reason or is something else going on?